docs/monitoring.md

```diff
@@ -253,10 +273,24 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Summary metrics of all tasks in the given stage attempt</td>
   </tr>
   <tr>
+    <td></td>
+    <td><code>?quantiles</code> summarize the metrics with the given quantiles.
+    Example: <br><code>?quantiles=0.01,0.5,0.99</code></td>
```

Comment:
Now that's interesting. in HTML 1.0-5.x it's like {{<p>}}: you don't insert a closing element in , though its permitted. See: [The br element](http://w3c.github.io/html/textlevel-semantics.html#the-br-element).

If the output goes straight to HTML then, leaving it open is the correct action. 

If, alternatively, its going to go through a process pipeline that generates XML and then things from it, then the closing tag matters.

What is the pipeline that jekyll implements?